examples/bq-log-alerting/example-foundation/README.md

```diff
@@ -0,0 +1,59 @@
+# Example: Deploy BigQuery Log Alerting solutiuon in terraform-example-foundation
+
+The solution deploys the BigQuery Log Alerting submodule in the logging project created in step `1 org` of the [terraform-example-foundation](https://github.com/terraform-google-modules/terraform-example-foundation).
```

Comment:
I think this should be more of a generic example for a user who wants to just take this solution and incorporate it. IMHO we should not have any dependencies on example-foundation to make use of this. If there are are prerequisite pieces necessary we can specify them and then link to example-foundation code as an example for setting up pre requisites. WDYT?

Reply:
Sounds good. 

The README can be generic regarding the prerequisites with a link to example-foundation code, It would help to show how to create the log sink for all the types of logs that will be needed by the use cases.

As long as users deploying it on example-foundation do not need to put extra effort to make it work I am good :)

Reply:
Totally agree, my expectation is that we are building this into `example-foundation` (by using this module) and there will be no extra effort from an end user perspective.